src/terminal/terminal.c

```diff
@@ -1458,22 +1428,38 @@ int guac_terminal_resize(guac_terminal* terminal, int width, int height) {
     /* Acquire exclusive access to terminal */
     guac_terminal_lock(terminal);
 
-    /* Calculate available text display area by character size */
-    int rows, columns;
-    calculate_rows_and_columns(terminal, height, width, &rows, &columns);
+   /* Set size of available screen area */
```

Comment:
Looks like a space was accidentally removed here (indentation is now misaligned).